.travis.yml

```diff
@@ -1,4 +1,4 @@
 language: node_js
 node_js:
-  - "5"
-  - "4"
+  - lts/*
+  - node
```

Comment:
LTS and latest are what most users are (and should?) be interested in, I figure.